unicornia/src/Blocks/custom/news/news-style.scss

```
@@ -0,0 +1,64 @@
+$block-news__content-wrap: (
```

```suggestion
$block-news: (
```

1 style === 1 map which should be named the same as the main selector. Then inside you can add a key for `content-wrap` for all styles specific to content-wrap.